articles/mysql/howto-data-encryption-portal.md

```diff
@@ -99,7 +99,8 @@ After Azure Database for MySQL is encrypted with a customer's managed key stored
 
    :::image type="content" source="media/concepts-data-access-and-security-data-encryption/show-revalidate-data-encryption.png" alt-text="Screenshot of Azure Database for MySQL, with revalidation step highlighted":::
 
-   You will have to give the key vault access to the new server.
+   You will have to give the key vault access to the new server. 
```

Comment:
Can we merge the two lines. 

You will have to give the key vault access to the new server. For more information, see [how to assign access policy on Key vault]( ../key-vault/general/assign-access-policy?tabs=azure-portal).

Reply:
changes been applied